src/pages/Cart/Cart.js

```diff
@@ -0,0 +1,198 @@
+import React from 'react';
+import CartListSection from './CartListSection';
+import CartTotalSection from './CartTotalSection';
+import CartButton from './CartButton';
+import CartTotalSectionFooter from './CartTotalSectionFooter';
+import { BASE_URL } from '../../config';
+import './Cart.scss';
+
+class Cart extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      cartData: [],
+      totalAmount: 0,
+      delivery: 0,
+    };
+  }
+
+  componentDidMount() {
+    fetch('/data/cartData.json')
+      .then(res => res.json())
+      .then(cartData => {
+        this.setState({
+          cartData: cartData,
+        });
+      });
+  }
+
+  calculateTotal = price => {
+    this.setState({
+      totalAmount: Number(this.state.cartData.price) + price,
+    });
+  };
+
+  quantityUpdate = idx => {
+    let copy = this.state.cartData.map((el, index) => {
+      if (idx === index) {
+        return { ...el, quantity: el.quantity + 1 };
+      } else {
+        return el;
+      }
+    });
+    this.setState({
+      cartData: copy,
+    });
+  };
+
+  quantitySubstract = idx => {
+    let copySub = this.state.cartData.map((el, index) => {
+      if (idx === index) {
+        return { ...el, quantity: el.quantity - 1 };
+      } else {
+        return el;
+      }
+    });
+    this.setState({
+      cartData: copySub,
+    });
+  };
+
+  orderFunction = idx => {
+    fetch(`${BASE_URL}/orders/cart`, {
+      headers: {
+        Authorization: localStorage.getItem('TOKKEN'),
+      },
+      method: 'POST',
+      body: JSON.stringify({
+        item_id: this.state.cartData[idx].item_id,
+        quantity: this.state.cartData[idx].quantity,
+      }),
+    })
+      .then(res => res.json())
+      .then(order => {
+        alert('성공');
+        //   this.props.push.history('/Payment');
+      });
```

Comment:
res로 들어온 데이터를 사용하지 않는다면 굳이 res.json()으로 javascript로 변환시킬 필요 없을 것 같네요!
```suggestion
      .then(() => {
        alert('성공');
        //   this.props.push.history('/Payment');
      });
```
나머지 부분도 마찬가지입니다!

Reply:
수정완료